modules/ROOT/pages/to-deploy-using-rest-connect.adoc

```diff
@@ -327,16 +327,47 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
 * `target`
 * `targetValue`
 
```

Comment:
why was this part deleted? https://docs.mulesoft.com/exchange/to-deploy-using-rest-connect#oauth2-in-design-center-for-rest-connect from the original docs?

Reply:
@liocastiglioni As noted in the PR, this section is outdated. The screen shots are outdated, there are missing steps, and it doesn't work as documented. If you can work with me to update it so it's current, we can do that.